### ISSUE TITLE
Where is the learnable parameter in your code before cosine similarity?

### ISSUE THREAD

**Issue by Jiawen-huang** [cyvius96/few-shot-meta-baseline#22]
in your paper:
Scaling the cosine similarity. Since cosine similarity has
the value range of [1; 1], when it is used to compute the
logits, it is important to scale the value before applying Softmax
function during training. We add a learnable parameter
r similar to recent works (Gidaris & Komodakis, 2018;
Qi et al., 2018; Oreshkin et al., 2018), that the predicted
probability in training becomes:

I can't find this learnable parameter r in your code . Your classifier seems to use cosine similarity directly?

**Comment by Jiawen-huang** [comment]
Thank you for your help. I have another question.
in train_meta_tiered.yaml, for the dataset tiered-imagenet, your train_batches=10,Why do you set it this way?
I tried to set train_batches=200 and found that the accuracy didn't improve in the meta-training stage。

**Comment by cyvius96** [comment]
You're welcome. There would be a quick "class-level overfitting", you can check the experiment part in our paper for details.
